index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const copyTextToClipboard = input => {
+const copyTextToClipboard = (input, target = document.body) => {
```

Comment:
I would prefer to make it an options-object, just in case we need to add additional options in the future.
```suggestion
const copyTextToClipboard = (input, {target = document.body} = {}) => {
```